container-images/common/Makefile

```
@@ -23,3 +23,11 @@ $(tools-requirements-files): requirements-tools-%.txt: requirements-tools.in
 	--generate-hashes \
 	--emit-index-annotation \
 	-o $@ $<
+
+requirements-rag.txt: requirements-rag.in
```

checkmake (0.3.2)

[warning] Target body for "requirements-rag.txt" exceeds allowed length of 5 lines (6).

(maxbodylength)